### PR TITLE
Adding position absolute to tooltip.

### PR DESCRIPTION
**Pull request by MayaUribe**
## Description

Updating Tooltip styles.

## Motivation and Context

In Publish the Tooltip stopped working, and adding the position absolute fixes it.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style and guidelines of this project. 
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] I have performed a self-review of my own code
- [x] I have tested my changes/additions in the latest Chrome, Firefox, and Safari.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] All new and existing tests passed.
- [ ] I have performed the accessibility audit of my UI changes according to the accessibility doc. 
- [ ] [Buffer Engineers] Someone from the Design team reviewed and approved my changes
- [ ] [Buffer Engineers] I have notified the BDS team of my changes in the #proj-design-system Slack channel

### CODE REVIEW

**Comment by anafilipadealmeida** [comment]
Hey @MayaUribe, out of curiosity, did you check what would happen if we updated the @reach/tooltip version to the latest? Would it solve the issue?
Also, instead of `npm run publish`, we could probably try `npm run beta:publish` and create a beta version of the UI just for Publish, since it's where the tooltip is failing (https://github.com/bufferapp/ui/blob/master/package.json#L151) cc/ @msanroman

**Comment by MayaUribe** [comment]
> Hey @MayaUribe, out of curiosity, did you check what would happen if we updated the @reach/tooltip version to the latest? Would it solve the issue?
> Also, instead of `npm run publish`, we could probably try `npm run beta:publish` and create a beta version of the UI just for Publish, since it's where the tooltip is failing (https://github.com/bufferapp/ui/blob/master/package.json#L151) cc/ @msanroman

Hey @anafilipadealmeida yes I tried updating the version, but it doesn't seem to fix the bug, wondering if I should bump the version either way? or leave it as it?

**Comment by anafilipadealmeida** [comment]
> > Hey @MayaUribe, out of curiosity, did you check what would happen if we updated the @reach/tooltip version to the latest? Would it solve the issue?
> > Also, instead of `npm run publish`, we could probably try `npm run beta:publish` and create a beta version of the UI just for Publish, since it's where the tooltip is failing (https://github.com/bufferapp/ui/blob/master/package.json#L151) cc/ @msanroman
> 
> Hey @anafilipadealmeida yes I tried updating the version, but it doesn't seem to fix the bug, wondering if I should bump the version either way? or leave it as is?

If it doesn't do any harm and it doesn't break anything I'd say to go for it as well 😄

File: src/documentation/markdown/GettingStarted/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [5.18.1] - 2019-12-04
+### Fixed
+- Added a position absolute for the Tooltip component, which stopped working in Publish
```
Comment (anafilipadealmeida) [inline src/documentation/markdown/GettingStarted/Changelog.md]: @MayaUribe, what do you think about removing the "which stopped working in Publish" part from the documentation? 🙂

**Comment by gmzjuliana** [comment]
> Hey @MayaUribe, out of curiosity, did you check what would happen if we updated the @reach/tooltip version to the latest? Would it solve the issue?
> Also, instead of `npm run publish`, we could probably try `npm run beta:publish` and create a beta version of the UI just for Publish, since it's where the tooltip is failing (https://github.com/bufferapp/ui/blob/master/package.json#L151) cc/ @msanroman

I think it's a great idea to create a beta version :) I know we're using the tooltips in Reply and would be great if Sol could confirm there's no breaking changes when she gets back from vacation :)

**Pull request merged**